DOCS-1980 Add IM getting started (#10936)
DataDog/documentation: content/en/monitors/incident_management/_index.md

    * Datadog deduplicates service names case-insensitively, so if you use "My Service" or "my service", only the manually added one is shown.
    * Datadog overrides APM service names in favor of the manually uploaded list.
    * Note that if the service is an APM service and no metrics are posted in the past seven days, it does not appear in the search results.
    * Further integrate with Datadog products and accurately assess service impact. The Services property field is automatically populated with APM services for customers using Datadog APM
* **Teams**: Populated from a CSV upload. Any values uploaded via CSV are only available within Incident Management for incident assessment purposes.

## Integrations

In addition to integrating with [Slack][3], Incident Management also integrates with:

- [PagerDuty][8] to send incident notifications to PagerDuty.
- [Jira][9] to create a Jira ticket for an incident.
- [Webhooks][10] to send incident notifications using webhooks (for example, [sending SMS to Twilio][11]).

## Ready to try it out?

> Work through an example workflow in our [Getting Started with Incident Management][12].

Reviewer: ```suggestion
Work through an example workflow in the [Getting Started with Incident Management][12] guide.
```

[1]: https://app.datadoghq.com/incidents
[2]: https://app.datadoghq.com/incidents/settings
[3]: /integrations/slack/?tab=slackapplicationbeta#using-the-slack-app

> [4]: https://docs.datadoghq.com/integrations/slack/?tab=slackapplicationus

Reviewer: ```suggestion
[4]: /integrations/slack/
```

[5]: https://app.datadoghq.com/incidents/ddslackapp
[6]: /getting_started/tagging/assigning_tags?tab=noncontainerizedenvironments#overview
[7]: /tracing/#2-instrument-your-application

> [8]: https://docs.datadoghq.com/integrations/pagerduty/#overview
> [9]: https://docs.datadoghq.com/integrations/jira/#overview
> [10]: https://docs.datadoghq.com/integrations/webhooks/#overview
> [11]: https://docs.datadoghq.com/integrations/webhooks/#sending-sms-through-twilio

Reviewer: ```suggestion
[8]: /integrations/pagerduty/
[9]: /integrations/jira/
[10]: /integrations/webhooks/
[11]: /integrations/webhooks/#sending-sms-through-twilio
```
